src/Sructs.c

```diff
@@ -0,0 +1,40 @@
+/*
+ Created by Elchairoy Meir on 17/11/2021.
+ Definition of table, column a cell structs.
+*/
+
+#define MAX_COLUMN_NAME_LEN 30
+#define MAX_TABLE_NAME_LEN 30
+
+enum types {Character = 0,Integer,Double,String}; /* All the possible types of data in each column/cell. */
```

Comment:
Do not call it "types". Call it "Type".
1. It should start with a capital char
2. It represents a single type.